tools/generate-helpers/pg-table-bindings/list.go

```diff
@@ -46,13 +46,12 @@ func init() {
 		&storage.K8SRoleBinding{}:     resources.K8sRoleBinding,
 		&storage.K8SRole{}:            resources.K8sRole,
 		// TODO: ROX-12750 Replace DebugLogs with Administration.
```

Comment:
What happened here? Can't spot the change :(

Reply:
L54-L55 -> L54
```
- // TODO: ROX-12750 Replace NetworkGraphConfig with Administration.
- &storage.NetworkGraphConfig{}: resources.NetworkGraphConfig,
+ &storage.NetworkGraphConfig{}: resources.Administration,
```
The rest is whitespace adjustments to align the resources between comments.